pkg/cmd/server/server_test.go

```diff
@@ -276,7 +275,7 @@ func TestRetryPolicy(t *testing.T) {
 		datastore.WithRequestHedgingEnabled(false),
 	)
 	if err != nil {
-		log.Fatalf("unable to start memdb datastore: %s", err)
```

Comment:
It was complaining here that `log.Fatalf` prevents `defer` blocks from running. `t.Fatalf` should be equivalent and also work around that.